### PR TITLE
Cumulative/Exceedance Graph: Added points option

### PR DESCRIPTION
**Pull request by N-thony**
Added points option. This solves issue #6111
@africanmathsinitiative/developers and @rdstern this is ready for review. Thanks.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@shadrackkibet please could you review? thanks

File: instat/dlgCumulativeDistribution.vb [hunk 1]
```diff
@@ -106,10 +109,10 @@ Public Class dlgCumulativeDistribution
         ucrChkCountsOnYAxis.SetText("Counts on Y Axis")
 
         ucrChkIncludePoints.SetText("Include Points")
-        ucrChkIncludePoints.SetParameter(New RParameter("geom", iNewPosition:=1), bNewChangeParameterValue:=False)
+        ' ucrChkIncludePoints.SetParameter(New RParameter(""))
```
Comment (shadrackkibet) [inline instat/dlgCumulativeDistribution.vb:112]: Remove?

File: instat/dlgCumulativeDistribution.vb [hunk 2]
```diff
@@ -159,10 +161,22 @@ Public Class dlgCumulativeDistribution
         clsStatECDFFunction.SetRCommand("stat_ecdf")
         clsStatECDFFunction.AddParameter("pad", "FALSE", iPosition:=0)
 
+        clsStatECDFPointFunction.SetPackageName("ggplot2")
+        clsStatECDFPointFunction.SetRCommand("stat_ecdf")
+        clsStatECDFPointFunction.AddParameter("pad", "FALSE", iPosition:=0)
+
+        clsBaseOperator.SetOperation("+")
+        clsBaseOperator.AddParameter("ggplot", clsRFunctionParameter:=clsRggplotFunction, iPosition:=0)
+        clsBaseOperator.AddParameter("labs", clsRFunctionParameter:=clsLabsFunction)
```
Comment (shadrackkibet) [inline instat/dlgCumulativeDistribution.vb:170]: Parameter position?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]

**Comment by shadrackkibet** [comment]
The output graph is okay. However, ` ggplot2::stat_ecdf(pad=FALSE) ` is added twice when Include points is not checked.

```
last_graph <- ggplot2::ggplot(data=Dagoretti_JKIA, mapping=ggplot2::aes(x=PRECIP, colour=station)) + ggplot2::stat_ecdf(pad=FALSE) + ggplot2::stat_ecdf(pad=FALSE) + theme_grey() + labs(y=NULL) + ggplot2::scale_y_continuous(breaks=seq(from=0, to=1, by=0.25))
```

**Comment by N-thony** [comment]
@shadrackkibet I solved the issues you raised a couple of hours ago. Could you confirm that the issues are resolved? Thanks.

**Comment by shadrackkibet** [comment]
Looks good. Thanks.

File: instat/dlgCumulativeDistribution.vb (hunk 2 above)
Comment (shadrackkibet) [reply instat/dlgCumulativeDistribution.vb:169]: I can't see this change.

Comment (N-thony) [reply instat/dlgCumulativeDistribution.vb:169]: Oh okay, thanks, doing it.

**Comment by N-thony** [comment]
@shadrackkibet I solved the issues you raised a couple of hours ago. Could you confirm that the issues are resolved? Thanks.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@shadrackkibet thank you for the fast review :). As far as I could see, all your comments were resolved. If I misunderstood, then please let me know
@rdstern I think this is now ready for your testing

**Review by rdstern — APPROVED** [review]
Great

**Pull request merged**